tmt-app/public/index.html

```diff
@@ -11,7 +15,7 @@
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <!--
-      manifest.json provides metadata used when your web app is installed on a
+      manifest.json provides metadata used when your web application is installed on a
```

Comment:
What was the need for this change?

Reply:
Nothing. I was trying to change the LF for the file but git does not register the change

Reply:
Can we leave the out of scope changes out?

Reply:
Also git should auto conver crlf(?) to LF if working on windows